bpf/maps_definition.h

```diff
@@ -79,4 +79,24 @@ struct {
     __uint(pinning, LIBBPF_PIN_BY_NAME);
 } peer_filter_map SEC(".maps");
 
+// HashMap to store ingress flowid to be able to retrieve them from kretprobe hook
+struct {
+    __uint(type, BPF_MAP_TYPE_HASH);
+    __uint(max_entries, 1 << 20); // Will take around 64MB of space.
```

Comment:
if I'm correct, the map content only lives during the xfrm call .... so maybe 64MB is over-sized? wdyt?

Reply:
its NO_PREALLOC map types i,e allocate as needed so there should be no issues with size right ?

Reply:
ok